user/TASK/shoot_task/shoot_task.c

```diff
@@ -210,15 +210,31 @@ static void shoot_reversed_control(Shoot_Motor_t *trigger_motor, Shoot_Motor_t *
 }
 
 
+static uint16_t shoot_count = 0;
+#define SHOOT_COUNT_MAX 1250
+#define SHOOT_COUNT_SWTICH 750
+
 /**
  * @brief Trigger motor and hopper motor continuously spinning to allow for rapid firing
  * @param Trigger motor and hopper motor structs
  * @retval None
  */
 static void shoot_rapid_control(Shoot_Motor_t *trigger_motor, Shoot_Motor_t *hopper_motor)
 {
-    //Hopper and trigger motor both spinning, used for clearing bullets
-    hopper_motor->speed_set = HOPPER_SPEED;
+    shoot_count++;
+    
+    if(shoot_count > SHOOT_COUNT_MAX){
+        shoot_count = 0;
+        hopper_motor->speed_set = HOPPER_SPEED;
+
+    }
+    if(shoot_count > SHOOT_COUNT_SWTICH){
+            hopper_motor->speed_set = HOPPER_SPEED * -0.5f;
+    }
+    else {
+        hopper_motor->speed_set = HOPPER_SPEED;;
```

Comment:
;

Reply:
How did this compile lol

Reply:
; just marks the end of statements, which can be blank, so you can have as many ; as you want